index.html

```diff
@@ -118,7 +117,7 @@ <h1 class="desktop">Data Dashboard Healthcare</h1>
 
     <div id="third-container" class="middle-1-container1">
       <h1 class="mobile1">Profesional Art Printing Data</h1>
```

Comment:
- [ ] Kindly change all `h1` tags except the first one in the headline section to `h2/h3...`. It's a good practice for a webpage to have only one `h1` tag and all subsequent headings can be h2, h3 in a hierarchical order.  For this project and for hierarchy purposes, you should have only one h1 tag on your entire page. please change all the headings for your sections to the `h2` tag 👍